src/components/HeroText/HeroText.module.css

```diff
@@ -1,172 +1,162 @@
 .position {
-  margin-top: 100px;
-  margin-left: 400px;
+    margin-top: 100px;
+    margin-left: 400px;
 }
 
 .hero {
-  font-size: calc(1em + 2vw);
-  font-family: 'Josefin Sans', sans-serif;
-  /* max-width: 50vw; */
+    font-size: calc(1em + 2vw);
+    font-family: 'Josefin Sans', sans-serif;
+    /* max-width: 50vw; */
 }
 
 .bodyText {
-  /* max-width: 35rem; */
-  position: relative;
-  line-height:1.8;
-  font-size: calc(0.8em + 0.3vw);
-  font-family: 'Poppins';
-  max-width: 40vw;
+    /* max-width: 35rem; */
+    position: relative;
+    line-height: 1.8;
+    font-size: calc(0.8em + 0.3vw);
+    font-family: 'Poppins';
+    max-width: 40vw;
 }
 
 .f {
-  position: relative;
-  width: calc(18px + 1vw);
+    position: relative;
+    width: calc(18px + 1vw);
 }
 
-.left{
-  position: relative;
-  left: 70px;
-  top: 40px;
-  width: calc(18px + 1vw);
-  transform: scaleY(-1) rotate(315deg);
+.left {
+    position: relative;
+    left: 70px;
+    top: 40px;
+    width: calc(18px + 1vw);
+    transform: scaleY(-1) rotate(315deg);
 }
 
-.right{
-  position: relative;
-  /* left: calc(2em + 10vw); */
-  top: calc(14em + 0.1vw);
-  width: calc(18px + 1vw);
-  transform: rotate(45deg);
+.right {
+    position: relative;
+    /* left: calc(2em + 10vw); */
+    top: calc(14em + 0.1vw);
+    width: calc(18px + 1vw);
+    transform: rotate(45deg);
 }
 
+
 /* @media screen and (max-width: 35em) {
   .bodyText {
     max-width: 100vw;
     width: 20em;
   }
 } */
 
-/* Mobile */
-@media (min-width: 0px) and (max-width: 767px){
-  .position {
-    margin: 0;
-  }
-  
-  .hero {
-    padding-left: 20px;
-  }
 
-  .bodyText {
-    max-width: none;
-    padding-left: 20px;
-    padding-right: 20px;
-    font-size: calc(0.8em + 1vw);
-  }
-
-  .f {
-    top: 15px;
-    left: 10px;
-  }
-  .left{
-    top: 30px;
-    left: 5px;
-  }
+/* Mobile */
 
-  .right{
-    left: 15px;
-    top: 30px;
-  }
+@media (min-width: 0px) and (max-width: 767px) {
+    .position {
+        margin: 0;
+    }
+    .hero {
+        padding-left: 20px;
+    }
+    .bodyText {
+        max-width: none;
+        padding-left: 20px;
+        padding-right: 20px;
+        font-size: calc(0.8em + 1vw);
+    }
+    .f {
+        top: 15px;
+        left: 10px;
+    }
+    .left {
+        top: 30px;
```

Comment:
similarly (not for this commit, but in general)- one may avoid such generic names like left/right